File: MM.Core/Entities/Fund.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MM.Core.Entities
{
    public class Fund : BaseEntity
    {
        public long MemberId { get; set; }
        public string Purpose { get; set; }
        public double Amount { get; set; }
        public bool IsDeduction { get; set; }
        public DateTime FundDate { get; set; }
        public virtual Member Member { get; set; }

    }
}

File: MessManagement.Web/ClientApp/src/app/Models/fund.ts
import { BaseEntity } from "./baseEntity";

export class Fund extends BaseEntity {
  memberId: number = 0;
  purpose: string = "";
  amount: number = 0;
  fundDate: any;
  isDeduction: boolean = false;
  memberFirstName: string = "";
  memberLastName?: string = "";
}

